analysis_options.yaml

```diff
@@ -12,4 +12,5 @@ linter:
     avoid_catches_without_on_clauses: false
     await_only_futures: true
     avoid_classes_with_only_static_members: false
-    prefer_single_quotes: true
\ No newline at end of file
+    prefer_single_quotes: true
+    lines_longer_than_80_chars: false
```

Comment:
Why change the standard flutter rule? The new formatting made the code way less readable and convenient to use.